### PR TITLE
Add limit on std_cdf input

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

File: contracts/option_pricing.cairo [hunk 1]
```diff
@@ -71,6 +71,11 @@ func std_normal_cdf{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_
         return (res=res);
     }
 
+    with_attr error_message("option_pricing.std_normal_cdf received X value higher than 8") {
+        let max_x_val = Math64x61.fromFelt(8);
+        assert_le(x, max_x_val);
+    }
+
```
Comment (MarekHauzr) [inline contracts/option_pricing.cairo:78]: what about lower then -8?

Comment (Chepelau) [reply contracts/option_pricing.cairo:78]: afaik the function always calculates only for positive X (that's why i put the assert after the first if statement), later the cdf value is handled with `res if is_positive(d) else 1-res` or sth like that

Comment (MarekHauzr) [reply contracts/option_pricing.cairo:78]: sorry, you are right.

Comment (MarekHauzr) [reply contracts/option_pricing.cairo:78]: missed it :/

**Pull request merged**